### PR TITLE
added some of julia's suggestions

### PR DESCRIPTION
**Pull request by joanibal**
## Purpose
Adds some of the suggested changes from Julia Weiss

## Expected time until merged
1 month

## Type of change

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (non-backwards-compatible fix or feature)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [x] Documentation update
- [ ] Maintenance update
- [ ] Other (please describe)

## Checklist

- [ ] I have run `flake8` and `black` to make sure the code adheres to PEP-8 and is consistently formatted
- [ ] I have run unit and regression tests which pass locally with my changes
- [ ] I have added new tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation

### CODE REVIEW

File: machAeroTutorials/opt_overview.rst [hunk 1]
```diff
@@ -12,6 +12,14 @@ There are many problems with this approach, including prohibitive computational
 Instead, we use a combination of analytic gradients, automatic differentiation, and the complex-step method to compute accurate gradients for our optimizations.
 We have spent a great deal of effort to make optimization a fairly straightforward and seamless process for the end user, so we hope you enjoy learning to use our tools!
 
+Overview of framework
+---------------------
+An optimizer is used via `pyOptSparse` to select the design variables for a cannidate design.
```
Comment (marcomangano) [inline machAeroTutorials/opt_overview.rst:17]: typo "candidate"

File: machAeroTutorials/opt_overview.rst [hunk 2]
```diff
@@ -12,6 +12,14 @@ There are many problems with this approach, including prohibitive computational
 Instead, we use a combination of analytic gradients, automatic differentiation, and the complex-step method to compute accurate gradients for our optimizations.
 We have spent a great deal of effort to make optimization a fairly straightforward and seamless process for the end user, so we hope you enjoy learning to use our tools!
 
+Overview of framework
+---------------------
+An optimizer is used via `pyOptSparse` to select the design variables for a cannidate design. 
+`pyGeo` warps the surface mesh according to the geometric design variables. 
+To create a volume mesh, `idwarp` propagates the deformation of the surface mesh into the volume mesh.
```
Comment (marcomangano) [inline machAeroTutorials/opt_overview.rst:19]: Would just remove `To create a volume mesh`, sounds wrong and the rest of the sentence is clear to me

File: machAeroTutorials/opt_overview.rst [hunk 3]
```diff
@@ -12,6 +12,14 @@ There are many problems with this approach, including prohibitive computational
...
+ADflow then solves for the performance of the design using that volume mesh. The variables using to define the aerodynamic analysis can also be selected by the optimizer (e.g. Mach number, AoA, etc.).
```
Comment (marcomangano) [inline machAeroTutorials/opt_overview.rst:20]: "used"?

File: machAeroTutorials/airfoilopt_ffd.rst [hunk 4]
```diff
@@ -51,7 +51,10 @@ The FFD box can now be set up.
     :end-before: # rst FFDBox2
 
 
-``nffd`` signifies the number of chordwise points. An empty FFD box is created.
+``nffd`` signifies the number of chordwise slices.
+The `2`'s in `(nffd, 2, 2, 3)` are the number of slices in the "spanwise" and "thickness" directions.
```
Comment (marcomangano) [inline machAeroTutorials/airfoilopt_ffd.rst:55]: I would rephrase like: "we pre-allocate an array of generic size (a,b,c,3) to set up an empty FFD box. In this example, a=nffd (number of chordwise sections), b=c=2 (number of spanwise and thickness-wise sections resepctively) and the final 3 is "fixed" as we are using 3D coordinates for each point"

not sure if this is clearer/nicer, up to you!

**Review by marcomangano — COMMENTED** [review]
Nice, see comments

File: machAeroTutorials/airfoilopt_ffd.rst (hunk 4 above)
Comment (joanibal) [reply machAeroTutorials/airfoilopt_ffd.rst:55]: I changed it to match what you suggested
